code/modules/mining/lavaland/megafauna_loot.dm

```diff
@@ -273,6 +273,7 @@
 	body_parts_covered = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
 	clothing_flags = THICKMATERIAL
 	resistance_flags = FIRE_PROOF|LAVA_PROOF|ACID_PROOF
+	transparent_protection = HIDEGLOVES|HIDESUITSTORAGE|HIDEJUMPSUIT|HIDESHOES
```

Comment:
any reason in particular why this is `transparent_protection` instead of `flags_inv` ? 

I feel like you shouldn't be able to see some of these through the HECK suit

Reply:
> any reason in particular why this is `transparent_protection` instead of `flags_inv` ?
> 
> I feel like you shouldn't be able to see some of these through the HECK suit

Suit storage is for oxy tank, the suit doesn't visually cover gloves or shoes, and I think you can see like 1 pixel of your jumpsuit's pants, can't test it rn.